.gitignore

```diff
@@ -1,2 +1,2 @@
 node_modules/
-
+stoof.txt
```

Comment:
This file doesn't seem to exist(maybe it did before), let's remove it from the gitignore

Reply:
will do